examples/Demo/FluentUI.Demo/Program.cs

```diff
@@ -54,6 +59,10 @@
 // Use the localization services
 app.UseRequestLocalization(new RequestLocalizationOptions().AddSupportedUICultures(["en", "fr"]));
 
+// API endpoint to expose MCP capabilities for WebAssembly clients
+app.MapGet("/api/mcp/capabilities", () => McpCapabilitiesData.GetSummary())
```

Comment:
Why did you add this API? The Blazor Demo project can be published using Server but also WASM mode to be hosted on a static website.